container/root/migration.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -e
+
+# Entrypoint to run migrations.
+# Instructions:
+# ./migration.sh -p <PATH_TO_SCRIPT> -s <SCRIPT>
+#
+# examle: ./migration.sh -p /app/migrations/scripts -s cool-migration-script.php
```

Comment:
wouldn't it be easier to provide a single param?

`./migration.sh /app/migrations/scripts/cool-migration-script.php`

Or better yet, we can call this file `command.sh` and assume that we can use it to run command in the container/app, not just migration(s).